14a_alpha_diversity.Rmd

```diff
@@ -109,6 +109,29 @@ se <- mia::estimateDiversity(se,
 head(colData(se)$shannon)
 ```
 
+Alpha diversities can be visualized with boxplot. Here, Shannon index is compared 
+between different sample type groups. Individual data points are visualized by 
+plotting them as points. With more data points, test `geom_jitter` function
+instead of `geom_point`.
+
+```{r visualize-shannon}
+library(ggplot2)
+ggplot(as.data.frame(colData(se)), aes(x = SampleType, y = shannon)) +
+  # Outliers are removed, because otherwise each data point would be plotted twice; 
+  # as an outlier of boxplot and as a point of dotplot.
+  geom_boxplot(outlier.shape = NA) + 
+  geom_point() + 
+  theme(text = element_text(size = 10))
+```
+To test, if these differences that can be seen in plot are statistically significant
+we do pairwise Wilcoxon test. It compares every group with each others to see if 
+their values differ. 
+
+```{r pairwise-wilcoxon}
+df <- as.data.frame(colData(se))
+pairwise.wilcox.test(df$shannon, df$SampleType,  p.adjust.method = "fdr")
+```
+
```

Comment:
Would be ideal to show the differnces in the figure itself using `ggsignif` or other suitable packages?

Reply:
Removed wilcoxon and added geom_signif